api/tests/integration/infrastructure/repositories/campaign-assessment-participation-repository_test.js

```diff
@@ -275,5 +276,45 @@ describe('Integration | Repository | Campaign Assessment Participation', functio
         expect(campaignAssessmentParticipation.lastName).to.equal('Doe');
       });
     });
+
+    context('When something wrong on campaign participations', function () {
+      it('throw NotFoundError when no existing campaign participation', async function () {
```

Comment:
`Suggestion`: context :- 'when something is wrong with a campaign participation'
it :- 'throws a NotFoundError when a campaign participation does not exist'